src/main/java/mate/academy/spring/model/dto/request/AuthenticationRequestDto.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.model.dto.request;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+public class AuthenticationRequestDto {
+    @NotNull
+    private String email;
+    @NotNull
+    @Size(min = 8, max = 20)
+    private String password;
```

Comment:
```suggestion
    private String email;
    private String password;
```
this is not our task here

Reply:
i`m started creating a passValidator and stoped later)